src/settings.js

```diff
@@ -1817,6 +1817,9 @@ var WASMFS = false;
 // [link]
 var SINGLE_FILE = false;
 
+// If true, does binary encoding of data instead of base64 encoding.
+var SINGLE_FILE_BINARY_ENCODE = true;
```

Comment:
If this is `true` by default why have a setting for this?

Is it because some folks can't handle UTF-8?  If so, it might be worth mentioning there.   e.g. "Disable this if you can't handle UTF-8 chars in the generated JS".

Reply:
The idea is that it would allow developers to pass `-sSINGLE_FILE_BINARY_ENCODE=0` to revert back to the previous base64 encoding, should issues arise.

Updated the text doc.